nodewatcher/modules/defaults/network_profile/models.py

```diff
@@ -36,5 +36,6 @@ class RegistryMeta:
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('wifi-uplink', _("Use a wireless interface for uplink")))
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('no-wifi-ap', _("Disable wireless AP")))
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('no-wifi-mesh', _("Disable wireless mesh")))
+registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('wifi-mesh-80211s', _("Use 802.11s for meshing")))
```

Comment:
And I am asking if we can have something ticked by default. Because then it would be easier to understand these options if you would have "enable mesh with ad-hoc" and "enable mesh with 802.11s".